packages/libwasmtime/libwasmtime.0.0.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+homepage: "https://github.com/LaurentMazare/ocaml-wasmtime"
+maintainer: "lmazare@gmail.com"
+bug-reports: "https://github.com/LaurentMazare/ocaml-wasmtime/issues"
+authors: [
+  "Laurent Mazare"
+]
+install: [
+  [
+    "sh"
+    "-c"
+    "test -d %{lib}%/libwasmtime/lib/libwasmtime.a || ( mkdir -p %{lib}%/libwasmtime && tar xf wasmtime-linux.tar.xz && mv -f wasmtime-*linux*/* %{lib}%/libwasmtime )"
+  ] { os = "linux" }
+  [
+    "sh"
+    "-c"
+    "test -d %{lib}%/libwasmtime/lib/libwasmtime.a || ( mkdir -p %{lib}%/libwasmtime && tar xf wasmtime-macos.tar.xz && mv -f wasmtime-*macos*/* %{lib}%/libwasmtime )"
+  ] { os = "macos" }
+]
+synopsis: "libwasmtime library package"
+description: """
+This is used by the wasmtime package to trigger the install of the
+libwasmtime library."""
+extra-source "wasmtime-linux.tar.xz" {
+  src: "https://github.com/bytecodealliance/wasmtime/releases/download/v0.21.0/wasmtime-v0.21.0-x86_64-linux-c-api.tar.xz"
+  checksum: "md5=fa7df1c3eb44a319e3e133c1ab764c24"
+}
+extra-source "wasmtime-macos.tar.xz" {
+  src: "https://github.com/bytecodealliance/wasmtime/releases/download/v0.21.0/wasmtime-v0.21.0-x86_64-macos-c-api.tar.xz"
+  checksum: "md5=8f4354d8f5c187a5daacd6867124d0b8"
+}
```

Comment:
```suggestion
}
available: [ os = "macos" | os = "linux" ]
```
untested, but you need something like this to prevent the package being selected on (e.g.) windows. Is there a libwasmtime on Windows out of interest?